### ISSUE TITLE
VOT2018 long-term benchmark "run_analysis" error

### ISSUE THREAD

**Issue by zhuangjunfei** [votchallenge/vot-toolkit#160]
Sorry to bother you again, I try ncc and static on VOT2018 long-term benchmark, and I have already obtain the results from two trackers, but when I run "run_analysis.m", I get an error like this
Subscripted assignment dimension mismatch.

Error in analyze_precision_recall>calculate_tpr_fscore (line 167)
certanty(i:i+size(groundtruth{s})-1, r) = values{s, r};

and I find that no "../VOT2018LT/vot-toolkit-master-LT/workspace/results/ncc/longterm/ballet/ballet_001_confidence.value" existence, the problem is how could I get “ballet_001_confidence.value"？

**Comment by lukacu** [comment]
The confidence file gets generated if the tracker reports confidence values, this is a requirement for the LT benchmark. The NCC example was modified to do this so I am not sure why did you not get any data here, did you run an old version of the example tracker?

**Comment by zhuangjunfei** [comment]
Yes! I have solved the problem by a new version of example tracker. During fixing process I find may be there is a small mistake in "./vot-toolkit-master/tracker/examples/matlab/ncc.m"
may be  line 15  " rand("seed", sum(clock)); " should be replaced by   rand('seed', sum(clock));
Thanks lukacu

**Comment by zhuangjunfei** [comment]
by the way, I wonder if you would be convenient to tell me when you test MDNet for long-term benchmark, do you pretrain MDNet with long-term dataset? or just train MDNet online?

**Comment by lukacu** [comment]
Thank you for the fix. Please do not go off-topic, this is an issue about the toolkit which I will now close. You can use our forum for general questions, but just to answer quickly: the rules of not using testing data in training apply for all VOT challenges.

**Comment by zhuangjunfei** [comment]
Thanks anyway
